inc/ThirdParty/Plugins/SimpleCustomCss.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace WP_Rocket\ThirdParty\Plugins;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Subscriber for compatibility with Simple Custom CSS plugin.
+ *
+ * @since  3.5.3
+ * @author Soponar Cristina
+ */
+class SimpleCustomCss implements Subscriber_Interface {
+	/**
+	 * Subscribed events for AMP.
+	 *
+	 * @since  3.5.3
+	 * @author Soponar Cristina
+	 * @inheritDoc
+	 */
+	public static function get_subscribed_events() {
+		$events = [];
+
+		if ( ! defined( 'SCCSS_FILE' ) ) {
+			return $events;
+		}
+
+		$events['wp_enqueue_scripts']           = [ 'cache_sccss', 98 ];
+		$events['update_option_sccss_settings'] = 'delete_cache_file';
+
+		return $events;
+	}
+
+	/**
+	 * Caches SCCSS code & remove the default enqueued URL
+	 *
+	 * @since 2.9
+	 * @author Remy Perona
+	 */
+	public function cache_sccss() {
+		$sccss = rocket_get_cache_busting_paths( 'sccss.css', 'css' );
+
+		if ( ! rocket_direct_filesystem()->exists( $sccss['filepath'] ) ) {
+			$this->create_cache_file( $sccss['bustingpath'], $sccss['filepath'] );
+		}
+
+		// Bailout if the SCCSS file could not be created.
+		if ( ! rocket_direct_filesystem()->exists( $sccss['filepath'] ) ) {
+			return;
+		}
```

Comment:
You can check that directly in the `create_cache_file` method and use a boolean return value for it